ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/blobs/BlobSidecarManagerImpl.java

```diff
@@ -79,16 +93,32 @@ public BlobSidecarManagerImpl(
   public SafeFuture<InternalValidationResult> validateAndImportBlobSidecar(
```

Comment:
I think we should find some better word for this and next method. We are not importing BlobSidecars here. validateAndPrepare maybe. Don't like either but import is definitely not a word for new flow from my view

Reply:
yeah... I was lazily naming it like the block counterpart but now is time to find a good one